docs/diagram-tech-arq.plantuml

```diff
@@ -12,22 +12,26 @@ left to right direction
 
 APIGateway(apiGateway, "API Gateway", "External API Patterns")
 ElasticKubernetesService(creditCrytoService, "Credit Crypto Service", "Saga Pattern")
-Blockchain(blockchainService, "Core Service", "Smart Contract")
+ElasticKubernetesService(blockchainService, "Core", "Block chain client")
 ManagedStreamingforApacheKafka(kafka, "Kafka", "Message Style Pattern")
 Database(databaseBlockchain, "Database", "Postgres Database")
-ElasticKubernetesService(erpService, "ERP", "Last mile service")
+Blockchain(blockchain, "blockchain", "blockchain")
+ElasticKubernetesService(exportService, "tx exporter", "Last mile service")
```

Comment:
this service will be in charge of transform tx payloads and send them to an external app or service.